scss/footer.scss

```diff
@@ -9,17 +9,35 @@
   .footer-logo {
     position: relative;
     margin-bottom: 20px;
+
+    @media only screen and (max-width: 1024px) {
```

Comment:
I think is better to separate the media at the end of the file so you dont have to write this line for each of the classes